user/e16/template.py

```diff
@@ -0,0 +1,61 @@
+pkgname = "e16"
+pkgver = "1.0.30"
+pkgrel = 0
+build_style = "gnu_configure"
+configure_args = [
+    "--prefix=/usr",
+    "--sysconfdir=/etc",
+    "--mandir=/usr/share/man",
+    "--localstatedir=/var",
+    "--enable-sound",
+    "--enable-sound-alsa",
+    "--enable-sound-pulse",
+]
+hostmakedepends = ["automake", "libtool", "pkgconf"]
+makedepends = [
+    "alsa-lib-devel",
+    "audiofile-devel",
+    "dbus-devel",
+    "fontconfig-devel",
+    "freetype-devel",
+    "gdk-pixbuf-devel",
+    "gettext-devel",
+    "glib-devel",
+    "imlib2-devel",
+    "libice-devel",
+    "libpulse-devel",
+    "libsm-devel",
+    "libsndfile-devel",
+    "libx11-devel",
+    "libxcomposite-devel",
+    "libxcursor-devel",
+    "libxdamage-devel",
+    "libxext-devel",
+    "libxfixes-devel",
+    "libxft-devel",
+    "libxinerama-devel",
+    "libxpm-devel",
+    "libxpresent-devel",
+    "libxrandr-devel",
+    "libxrender-devel",
+    "libxres-devel",
+    "libxxf86vm-devel",
+    "pango-devel",
+    "xbitmaps",
+    "xorgproto",
+]
+depends = ["desktop-file-utils", "python"]
+pkgdesc = "Themed window manager for X11"
+maintainer = "rane <rane+chimera@junkyard.systems>"
+license = "BSD-2-Clause"
+url = "https://enlightenment.org/e16"
+source = f"https://downloads.sourceforge.net/enlightenment/e16/{pkgver}/e16-{pkgver}.tar.gz"
+sha256 = "b8b7748a2c48c4c7c9758d9ad12b14f566d2bec38f2eda533e6d874f5ce9074c"
+
+
+def init_install(self):
+    self.make_install_args += ["DESTDIR=" + str(self.chroot_destdir), "install"]
```

Comment:
this is the default install for gnu_configure, pretty sure it works without this

Reply:
OK thanks, that's really helpful to know. I changed it and it worked without any issues.